homeassistant/components/reolink/strings.json

```diff
@@ -233,6 +233,13 @@
           "off": "[%key:component::binary_sensor::entity_component::gas::state::off%]",
           "on": "[%key:component::binary_sensor::entity_component::gas::state::on%]"
         }
+      },
+      "sleeping": {
+        "name": "Sleeping",
```

Comment:
In my opinion, we should change the name as it's not clear what an entity named "Sleeping" with the state "Active" means. It could mean that the camera is currently sleeping, but from the options below it probably should mean the opposite

Reply:
True, do you have a good suggestion for the entity name/states?

Reply:
Maybe "Sleep status" as name and "awake" and "sleeping" as the two states?